# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/qcad/cstring_convert.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./qcadlib/src/filters/rs_filterdxf.cpp.orig	2005-03-18 10:56:35.485734958 +0100
+++ ./qcadlib/src/filters/rs_filterdxf.cpp	2005-03-18 11:07:21.593857587 +0100
@@ -509,7 +509,7 @@
         lss = RS2::Exact;
     }
 
-	mtext+=data.text;
+	mtext+=data.text.c_str();
 
     RS_String locallyEncoded = mtext;
     RS_String enc = RS_System::getEncoding(
